hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestWatchForCommit.java

```diff
@@ -129,6 +129,11 @@ private String getKeyName() {
     return UUID.randomUUID().toString();
   }
 
+  private XceiverClientMetrics getXceiverClientMetrics() {
+    RpcClient rpc = (RpcClient)client.getObjectStore().getClientProxy();
+    return rpc.getXceiverClientManager().getMetrics();
+  }
+
```

Comment:
Wondering if we could move this method so that we don't have to duplicate code in 4 test classes here.
Not a deal breaker but I am just tracking Sonar code duplication stats :)

Reply:
Done in the next patch